{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://github.com/pytest-dev/pytest-faulthandler",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-1.3.1.tar.gz",
    "subdirs": [
      "win-64"
    ],
    "summary": "py.test plugin that activates the fault handler module for tests",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "1.3.1"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main"
  ],
  "package": "pytest-faulthandler-1.3.1-py27_0.tar.bz2",
  "repodata": {
    "build": "py27_0",
    "build_number": 0,
    "depends": [
      "pytest",
      "python 2.7*"
    ],
    "license": "MIT",
    "md5": "9266a05fb9f4609d301ed7fa1446b25e",
    "name": "pytest-faulthandler",
    "sha256": "761303849e1f3c218cbf5a7c3a7b979b8cd701fdf890e6d8de5a3077b2c93924",
    "size": 4549,
    "subdir": "win-64",
    "version": "1.3.1"
  },
  "repodata_version": 1,
  "subdir": "win-64",
  "url": "https://conda.anaconda.org/conda-forge/win-64/pytest-faulthandler-1.3.1-py27_0.tar.bz2"
}